tests/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think it's a good habit too always add the utf8 encoding header everywhere. It makes code much more consistent